docs/userGuide/syntax/mathformulae.mbdf

```diff
@@ -0,0 +1,60 @@
+## Math Formulae
+Markbind supports typesetting math equations with [KaTeX](https://katex.org).
+
+<box type="info">
+
+Insert inline equations by enclosing them in round brackets `\( ... \)`.
+
+Insert block equations by enclosing them in square brackets `\[ ... \]`.
```

Comment:
Should this be `display equations` instead? It seems to be referred in that manner quite often.